.circleci/config.yml

```
@@ -1597,6 +1606,8 @@ workflows:
           test_list: git diff origin/develop...HEAD --name-only -- './test/**/*.t.sol' | sed 's|packages/contracts-bedrock/||'
           test_timeout: 1h
           test_fuzz_runs: 10000
+          test_invariant_runs: 256
+          test_invariant_depth: 256
```

similar idea, I'm just choosing numbers that keeps runs*depth the same, but feel free to tweak
```suggestion
          test_invariant_runs: 128
          test_invariant_depth: 512
```